### PR TITLE
Add docs about the App preview feature

### PR DESCRIPTION
**Pull request by helinanever**
Introduced a new page in the Configuration > Testing section for both yaml and Flutter workflow editor views.

### CODE REVIEW

**Comment by helinanever** [comment]
Since the info on this page is common for yaml and Flutter workflow editor users, I've added all the content into a single partial file and included it in respective category pages. The downside of this approach is that we won't be able to show the table of contents for the page. We could split the content file into multiple partials or just duplicate full content in both categories to work around this.

File: content/partials/app-preview-builds.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+---
+
+Codemagic allows you to preview your `.app` artifact on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical machine.
```
Comment (mikhail-tokarev) [inline content/partials/app-preview-builds.md:4]:
```suggestion
Codemagic allows you to preview your unsigned application (`.app` artifact) on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical phone.
```

File: content/partials/app-preview-builds.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Running preview builds
+
+You can preview any `.app` artifact built in Codemagic that targets the iOS simulator. For such artifacts, there is a **Quick Launch** button available next to the artifact name on the build overview page.
+
+Clicking **Quick Launch** displays a configuration popup for selecting the simulator device and runtime. Once you click **Start**, Codemagic starts a preview build during which the simulator is booted and the app is installed on the simulator. 
+
+When the simulator is ready for use, you'll see the URL to access the simulator printed in the logs under the "Running iOS simulator" step as well as a link to open the simulator on the left panel of the build overview page. The simulator session remains active for a maximum of **20 minutes** or until you cancel the build. 
+
+Once the session ends, the virtual machine running the build is wiped clean and destroyed.
```
Comment (mikhail-tokarev) [inline content/partials/app-preview-builds.md:18]:
```suggestion
Once the session ends you can re-run it if needed.
```

File: content/partials/app-preview-builds.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## Build minutes usage
+
+Preview builds run on a macOS VM and consume build minutes at the same per-minute rate as regular builds. If your team is on a Pay as you go plan, the minutes used for app previews will be added to your macOS minutes count.
```
Comment (mikhail-tokarev) [inline content/partials/app-preview-builds.md:22]:
```suggestion
Preview builds run on a macOS and consume build minutes at the same per-minute rate as regular builds. If your team is on a Pay as you go plan, the minutes used for app previews will be added to your macOS minutes count.
```

**Review by mikhail-tokarev — APPROVED** [review]
left some minor suggestions and removed `VM` in one place.
overall LGTM

File: content/partials/app-preview-builds.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+---
+
+Codemagic allows you to preview your unsigned `.app` artifact on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical device.
```
Comment (priitlatt) [inline content/partials/app-preview-builds.md:4]: Apps can be signed too, it is just unlikely that anyone would go through the trouble.

```suggestion
Codemagic allows you to preview your `.app` artifact on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical device.
```

File: content/partials/app-preview-builds.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+Note that this feature is available for **teams** on the Pay as you go billing plan by default and can be enabled for teams on the annual plan on request.
+{{</notebox>}}
+
+## Running preview builds
+
+You can preview any `.app` artifact built in Codemagic that targets the iOS simulator. For such artifacts, there is a **Quick Launch** button available next to the artifact name on the build overview page.
+
+Clicking **Quick Launch** displays a configuration popup for selecting the simulator device and runtime. Once you click **Start**, Codemagic starts a preview build during which the simulator is booted and the app is installed on the simulator. 
+
+When the simulator is ready for use, you'll see the URL to access the simulator printed in the logs under the "Running iOS simulator" step as well as a link to open the simulator on the left panel of the build overview page. The simulator session remains active for a maximum of **20 minutes** or until you cancel the build.
```
Comment (priitlatt) [inline content/partials/app-preview-builds.md:16]: As the link on sidebar is far more user friendly I'd try to highlight it over the URL in logs, which isn't even formatted as a proper href, rendering it difficult to use. For starters I presume that even documenting the sidebar link/button first would be enough.

**Review by priitlatt — APPROVED** [review]
Approving with absolutely minor remarks.

File: content/partials/app-preview-builds.md (hunk 5 above)
Comment (helinanever) [reply content/partials/app-preview-builds.md:16]: As the sidebar link may also be hard to spot as it appears there only when the simulator becomes available, I think it's still a good idea to point out that the URL is also available in the logs.

**Pull request merged**